source/release-notes.txt

```diff
@@ -10,6 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Enable "Modern Dark Mode" by default
+- Surface type warnings (:issue:`COMPASS-6543`)
+- Combine array fields into one in the import CSV preview (:issue:`COMPASS-6766`)
+- Remove export feature flag, use new export, remove old code (:issue:`COMPASS-6582`)
+- Add settings to sidebar menus (:issue:`COMPASS-6796`)
+- Show password only when user focuses on input (:issue:`COMPASS-6161`)
+- Add autocomplete support for $percentile, $median ans $$USER_ROLES (:issue:`COMPASS-6780`) (:issue:`COMPASS-6781`)
+- Toggle state of "Edit connection string" in connection form is now judged also based on the new global preference
+- Update libmongocrypt and driver, re-enable QE tests in 7.0, add test to ensure 6.0 QE data can be read 6601 (:issue:`COMPASS-6602`)
```

Comment:
I don't know what "read 6601" means. Can we phrase this bullet in a way that will mean something to users, or remove it entirely if it's internal?

Reply:
I think this was a typo and should refer to the COMPASS-6601 ticket, updated.